lib/magicbell/client.rb

```diff
@@ -3,7 +3,8 @@ class Client
     class HTTPError < StandardError
       attr_accessor :response_status,
                     :response_headers,
-                    :response_body
+                    :response_body,
```

Comment:
Are you going to add some specs for these?

Reply:
added